CRITICAL - 2025-06-09 10:10:21 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "Admin/diskon.php"
[Method: GET, Route: diskon]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('Admin/diskon.php')
 2 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/diskon.php', [], true)
 3 APPPATH\Controllers\AdminController.php(19): view('Admin/diskon.php')
 4 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->diskon()
 5 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:16:11 --> Error: Class "App\Controllers\UserModel" not found
[Method: POST, Route: register/save]
in APPPATH\Controllers\AuthController.php on line 30.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:17:27 --> Error: Class "App\Controllers\UserModel" not found
[Method: POST, Route: register/save]
in APPPATH\Controllers\AuthController.php on line 31.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:17:36 --> Error: Class "App\Controllers\UserModel" not found
[Method: POST, Route: register/save]
in APPPATH\Controllers\AuthController.php on line 31.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:17:50 --> Error: Class "App\Models\User_Model" not found
[Method: POST, Route: register/save]
in APPPATH\Controllers\AuthController.php on line 31.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:18:00 --> Error: Class "App\Models\User_Model" not found
[Method: POST, Route: register/save]
in APPPATH\Controllers\AuthController.php on line 31.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:18:39 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'tibra', 3360, '', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-09 12:18:39 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:19:00 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'tibra', 3360, '', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-09 12:19:00 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:19:20 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'tibra', 3360, '', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-09 12:19:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:19:26 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'tibra', 3360, '', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-09 12:19:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:20:22 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'tibra', 3360, '', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-09 12:20:22 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:21:59 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'tibra', 3360, '', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-09 12:21:59 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:22:40 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'tibra', 3360, '', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-09 12:22:40 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:23:08 --> mysqli_sql_exception: Table 'tibra.users' doesn't exist in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-06-09 12:23:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'tibra.users' doesn't exist
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:23:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'tibra.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'test\', \'test@email.com\', \'$2y$10$V6EHLh3NTe3g4.BVyhTfQeP9PZyRbusM7o0IwgjC66BW8JfoRJquO\', \'Customer\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'test\', \'test@email.com\', \'$2y$10$V6EHLh3NTe3g4.BVyhTfQeP9PZyRbusM7o0IwgjC66BW8JfoRJquO\', \'Customer\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:23:09 --> [Caused by] mysqli_sql_exception: Table 'tibra.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'test\', \'test@email.com\', \'$2y$10$V6EHLh3NTe3g4.BVyhTfQeP9PZyRbusM7o0IwgjC66BW8JfoRJquO\', \'Customer\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'test\', \'test@email.com\', \'$2y$10$V6EHLh3NTe3g4.BVyhTfQeP9PZyRbusM7o0IwgjC66BW8JfoRJquO\', \'Customer\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'test\', \'test@email.com\', \'$2y$10$V6EHLh3NTe3g4.BVyhTfQeP9PZyRbusM7o0IwgjC66BW8JfoRJquO\', \'Customer\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:23:17 --> mysqli_sql_exception: Table 'tibra.users' doesn't exist in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-06-09 12:23:17 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'tibra.users' doesn't exist
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:23:17 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'tibra.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$uF1QNIVSspJ79P7m4ljsH.fW3/MBH7A8/dcDja5rZzcsURh2Zh1cm\', \'Customer\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$uF1QNIVSspJ79P7m4ljsH.fW3/MBH7A8/dcDja5rZzcsURh2Zh1cm\', \'Customer\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:23:17 --> [Caused by] mysqli_sql_exception: Table 'tibra.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$uF1QNIVSspJ79P7m4ljsH.fW3/MBH7A8/dcDja5rZzcsURh2Zh1cm\', \'Customer\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$uF1QNIVSspJ79P7m4ljsH.fW3/MBH7A8/dcDja5rZzcsURh2Zh1cm\', \'Customer\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$uF1QNIVSspJ79P7m4ljsH.fW3/MBH7A8/dcDja5rZzcsURh2Zh1cm\', \'Customer\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:23:28 --> Error connecting to the database: mysqli_sql_exception: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:202
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(202): mysqli->real_connect('localhost', 'root', '', 'tibra', 3360, '', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No connection could be made because the target machine actively refused it in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:246
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(421): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#3 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#4 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#5 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#6 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#7 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-09 12:23:28 --> CodeIgniter\Database\Exceptions\DatabaseException: Unable to connect to the database.
Main connection [MySQLi]: No connection could be made because the target machine actively refused it
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 467.
 1 SYSTEMPATH\Database\BaseConnection.php(620): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 3 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 4 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 5 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 6 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
ERROR - 2025-06-09 12:24:20 --> mysqli_sql_exception: Table 'tibra.users' doesn't exist in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tibra\app\Controllers\AuthController.php(38): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#15 {main}
CRITICAL - 2025-06-09 12:24:20 --> CodeIgniter\Database\Exceptions\DatabaseException: Table 'tibra.users' doesn't exist
[Method: POST, Route: register/save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:24:20 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Table 'tibra.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$aoxp4JpL1qtOnR1SyRH15.xoIMhCYzifB.RDIO7xyeX52p2yqLHcS\', \'Customer\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$aoxp4JpL1qtOnR1SyRH15.xoIMhCYzifB.RDIO7xyeX52p2yqLHcS\', \'Customer\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 12:24:20 --> [Caused by] mysqli_sql_exception: Table 'tibra.users' doesn't exist
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$aoxp4JpL1qtOnR1SyRH15.xoIMhCYzifB.RDIO7xyeX52p2yqLHcS\', \'Customer\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$aoxp4JpL1qtOnR1SyRH15.xoIMhCYzifB.RDIO7xyeX52p2yqLHcS\', \'Customer\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (\'123\', \'test@email.com\', \'$2y$10$aoxp4JpL1qtOnR1SyRH15.xoIMhCYzifB.RDIO7xyeX52p2yqLHcS\', \'Customer\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `users` (`fullname`, `email`, `password`, `role`) VALUES (:fullname:, :email:, :password:, :role:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\AuthController.php(38): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AuthController->saveRegister()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AuthController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 12:24:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:31:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:31:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 12:31:08 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "dashboard.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('dashboard.php')
 2 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 3 APPPATH\Config\Routes.php(30): view('dashboard')
 4 SYSTEMPATH\CodeIgniter.php(875): CodeIgniter\Router\RouteCollection->{closure}()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 12:31:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 12:31:45 --> CodeIgniter\View\Exceptions\ViewException: Invalid file: "dashboard.php"
[Method: GET, Route: dashboard]
in SYSTEMPATH\Exceptions\FrameworkException.php on line 39.
 1 SYSTEMPATH\View\View.php(212): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('dashboard.php')
 2 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('dashboard', [], true)
 3 APPPATH\Config\Routes.php(30): view('dashboard')
 4 SYSTEMPATH\CodeIgniter.php(875): CodeIgniter\Router\RouteCollection->{closure}()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 12:34:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:37:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:38:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:39:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:39:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:39:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:39:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:39:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:40:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:40:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:41:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:41:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:41:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:41:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:41:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:41:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:42:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:42:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:43:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:43:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:43:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:44:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:46:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:47:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:48:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:49:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:50:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:50:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:50:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:50:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:50:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:50:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:51:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:51:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:51:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:51:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:52:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:52:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:52:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:52:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:55:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 12:55:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:11:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:12:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:15:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:15:02 --> ErrorException: Undefined variable $produk
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 229.
 1 APPPATH\Views\Admin\produk.php(229): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $produk', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 229)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(18): view('Admin/produk.php')
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 13:15:28 --> ErrorException: Undefined property: App\Controllers\AdminController::$produkModel
[Method: GET, Route: produk]
in APPPATH\Controllers\AdminController.php on line 18.
 1 APPPATH\Controllers\AdminController.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\AdminController::$produkModel', 'C:\\xampp\\htdocs\\tibra\\app\\Controllers\\AdminController.php', 18)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 13:15:44 --> ErrorException: Undefined property: App\Controllers\AdminController::$ProdukModel
[Method: GET, Route: produk]
in APPPATH\Controllers\AdminController.php on line 18.
 1 APPPATH\Controllers\AdminController.php(18): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\AdminController::$ProdukModel', 'C:\\xampp\\htdocs\\tibra\\app\\Controllers\\AdminController.php', 18)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 13:16:14 --> ErrorException: Undefined property: App\Controllers\AdminController::$ProdukModel
[Method: GET, Route: produk]
in APPPATH\Controllers\AdminController.php on line 19.
 1 APPPATH\Controllers\AdminController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\AdminController::$ProdukModel', 'C:\\xampp\\htdocs\\tibra\\app\\Controllers\\AdminController.php', 19)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 13:16:16 --> ErrorException: Undefined property: App\Controllers\AdminController::$ProdukModel
[Method: GET, Route: produk]
in APPPATH\Controllers\AdminController.php on line 19.
 1 APPPATH\Controllers\AdminController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\AdminController::$ProdukModel', 'C:\\xampp\\htdocs\\tibra\\app\\Controllers\\AdminController.php', 19)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 13:16:39 --> ErrorException: Undefined property: App\Controllers\AdminController::$ProdukModel
[Method: GET, Route: produk]
in APPPATH\Controllers\AdminController.php on line 19.
 1 APPPATH\Controllers\AdminController.php(19): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: App\\Controllers\\AdminController::$ProdukModel', 'C:\\xampp\\htdocs\\tibra\\app\\Controllers\\AdminController.php', 19)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:18:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:20:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:20:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:20:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:21:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:21:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:21:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:22:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:22:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:22:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:22:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:25:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:29:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-09 13:29:25 --> mysqli_sql_exception: Column 'photo_produk' cannot be null in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\tibra\system\BaseModel.php(754): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\tibra\app\Controllers\AdminController.php(68): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AdminController->store()
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
#11 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-09 13:29:25 --> CodeIgniter\Database\Exceptions\DatabaseException: Column 'photo_produk' cannot be null
[Method: POST, Route: store]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (:nama_produk:, :harga_produk:, :deskripsi_produk:, :merk_produk:, :type_produk:, :tahun_pembuatan:, :photo_produk:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\AdminController.php(68): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->store()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 13:29:25 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Column 'photo_produk' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'MSI\', \'210000\', \'2131q\', \'dasda\', \'Keyboard\', \'2100\', NULL)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'MSI\', \'210000\', \'2131q\', \'dasda\', \'Keyboard\', \'2100\', NULL)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (:nama_produk:, :harga_produk:, :deskripsi_produk:, :merk_produk:, :type_produk:, :tahun_pembuatan:, :photo_produk:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\AdminController.php(68): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->store()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 13:29:25 --> [Caused by] mysqli_sql_exception: Column 'photo_produk' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'MSI\', \'210000\', \'2131q\', \'dasda\', \'Keyboard\', \'2100\', NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'MSI\', \'210000\', \'2131q\', \'dasda\', \'Keyboard\', \'2100\', NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'MSI\', \'210000\', \'2131q\', \'dasda\', \'Keyboard\', \'2100\', NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (:nama_produk:, :harga_produk:, :deskripsi_produk:, :merk_produk:, :type_produk:, :tahun_pembuatan:, :photo_produk:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\AdminController.php(68): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->store()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:29:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:29:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:30:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:30:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:31:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:32:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:32:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:32:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:33:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:34:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:35:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:36:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:39:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:41:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:41:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:42:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:42:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:48:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:48:01 --> ErrorException: Undefined array key "type_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 405.
 1 APPPATH\Views\Admin\produk.php(405): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "type_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 405)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:48:19 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:48:19 --> ErrorException: Undefined array key "tipe_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 238.
 1 APPPATH\Views\Admin\produk.php(238): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "tipe_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 238)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:48:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:48:32 --> ErrorException: Undefined array key "type_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 405.
 1 APPPATH\Views\Admin\produk.php(405): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "type_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 405)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:49:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:49:46 --> ErrorException: Undefined array key "type_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 405.
 1 APPPATH\Views\Admin\produk.php(405): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "type_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 405)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:50:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:50:17 --> ErrorException: Undefined array key "type_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 406.
 1 APPPATH\Views\Admin\produk.php(406): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "type_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 406)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:50:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:50:18 --> ErrorException: Undefined array key "type_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 406.
 1 APPPATH\Views\Admin\produk.php(406): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "type_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 406)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:51:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:51:09 --> ErrorException: Undefined array key "type_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 407.
 1 APPPATH\Views\Admin\produk.php(407): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "type_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 407)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:51:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:51:28 --> ErrorException: Undefined array key "photo_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 430.
 1 APPPATH\Views\Admin\produk.php(430): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "photo_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 430)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:52:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:53:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:54:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:55:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:56:14 --> ArgumentCountError: Too few arguments to function App\Controllers\AdminController::update(), 0 passed in C:\xampp\htdocs\tibra\system\CodeIgniter.php on line 933 and exactly 1 expected
[Method: POST, Route: update]
in APPPATH\Controllers\AdminController.php on line 83.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->update()
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:58:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:58:44 --> ErrorException: Undefined array key "id_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 378.
 1 APPPATH\Views\Admin\produk.php(378): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 378)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:58:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:58:45 --> ErrorException: Undefined array key "id_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 378.
 1 APPPATH\Views\Admin\produk.php(378): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 378)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:58:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:58:47 --> ErrorException: Undefined array key "id_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 378.
 1 APPPATH\Views\Admin\produk.php(378): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 378)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:58:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-09 13:58:50 --> ErrorException: Undefined array key "id_produk"
[Method: GET, Route: produk]
in APPPATH\Views\Admin\produk.php on line 378.
 1 APPPATH\Views\Admin\produk.php(378): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key "id_produk"', 'C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php', 378)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tibra\\app\\Views\\Admin\\produk.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('Admin/produk.php', [], true)
 5 APPPATH\Controllers\AdminController.php(28): view('Admin/produk.php', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->produk()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 13:59:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:59:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:59:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:59:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 13:59:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:01:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:01:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:01:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-09 14:05:18 --> mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY' in C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tibra\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `pr...', 0)
#1 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pr...')
#2 C:\xampp\htdocs\tibra\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pr...')
#3 C:\xampp\htdocs\tibra\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pr...', Array, false)
#4 C:\xampp\htdocs\tibra\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tibra\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tibra\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\tibra\system\BaseModel.php(754): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\tibra\app\Controllers\AdminController.php(68): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\tibra\system\CodeIgniter.php(933): App\Controllers\AdminController->store()
#10 C:\xampp\htdocs\tibra\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
#11 C:\xampp\htdocs\tibra\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\tibra\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\tibra\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\tibra\public\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\tibra\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-09 14:05:18 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
[Method: POST, Route: store]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (:nama_produk:, :harga_produk:, :deskripsi_produk:, :merk_produk:, :type_produk:, :tahun_pembuatan:, :photo_produk:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\AdminController.php(68): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->store()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 14:05:18 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'Terong\', \'500000\', \'Ulah\', \'200\', \'Mouse\', \'4001\', \'1749477918_b511948d2ced7544d833.jpg\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'Terong\', \'500000\', \'Ulah\', \'200\', \'Mouse\', \'4001\', \'1749477918_b511948d2ced7544d833.jpg\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (:nama_produk:, :harga_produk:, :deskripsi_produk:, :merk_produk:, :type_produk:, :tahun_pembuatan:, :photo_produk:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\AdminController.php(68): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->store()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
CRITICAL - 2025-06-09 14:05:18 --> [Caused by] mysqli_sql_exception: Duplicate entry '0' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'Terong\', \'500000\', \'Ulah\', \'200\', \'Mouse\', \'4001\', \'1749477918_b511948d2ced7544d833.jpg\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'Terong\', \'500000\', \'Ulah\', \'200\', \'Mouse\', \'4001\', \'1749477918_b511948d2ced7544d833.jpg\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (\'Terong\', \'500000\', \'Ulah\', \'200\', \'Mouse\', \'4001\', \'1749477918_b511948d2ced7544d833.jpg\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `produk` (`nama_produk`, `harga_produk`, `deskripsi_produk`, `merk_produk`, `type_produk`, `tahun_pembuatan`, `photo_produk`) VALUES (:nama_produk:, :harga_produk:, :deskripsi_produk:, :merk_produk:, :type_produk:, :tahun_pembuatan:, :photo_produk:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(754): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\AdminController.php(68): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\AdminController->store()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\AdminController))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\tibra\\public\\index.php')
DEBUG - 2025-06-09 14:07:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:07:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:07:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:07:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:07:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:07:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:07:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:08:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:10:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:10:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:10:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:10:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:10:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:10:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:10:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-09 14:10:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
